submissions/iva-stasia/tiny-js-world/index.js

```diff
@@ -0,0 +1,51 @@
+import { print } from './js/lib.js';
+
+const dog = {
+   species: 'dog',
+   name: 'Ozzy',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Woof-woof!',
+};
+
+const cat = {
+   species: 'cat',
+   name: 'Debbie',
+   gender: 'female',
+   legs: 4,
+   hands: 0,
+   saying: 'Meeeow!',
+};
+
+const women = {
+   species: 'human',
+   name: 'Gwen',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Hi there!'
+};
+
+const man = {
+   species: 'human',
+   name: 'David',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'What\'s up?'
+};
+
+const catWomen = {
+   species: 'cat-human',
+   name: 'Dita',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: cat.saying
```

Comment:
Your catwoman's saying should change when cat's saying changes. Your's does not. You only take the initial value of the `cat.saying`.

Reply:
Now I understand that it only works once when I create the object.
Changed it.